Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -793,18 +809,6 @@ void Scene_c3t3_item::draw(CGAL::Three::Viewer_interface* viewer) const {
     d->spheres->setPlane(this->plane());
   }
 
-  vaos[Scene_c3t3_item_priv::Grid]->bind();
-  d->program = getShaderProgram(PROGRAM_WITHOUT_LIGHT);
-  attribBuffers(viewer, PROGRAM_WITHOUT_LIGHT);
-  d->program->bind();
-  d->program->setAttributeValue("colors", QColor(Qt::black));
-  QMatrix4x4 f_mat;
-  for (int i = 0; i<16; i++)
-    f_mat.data()[i] = d->frame->matrix()[i];
-  d->program->setUniformValue("f_matrix", f_mat);
-  viewer->glDrawArrays(GL_LINES, 0, static_cast<GLsizei>(d->positions_grid.size() / 3));
-  d->program->release();
-  vaos[Scene_c3t3_item_priv::Grid]->release();
```

Comment:
@maxGimeno I did fix the drawing of the grid **after** the rest in the PR #1468, but you reverted my change in this PR., probably during a rebase. Please be careful when you resolve conflicts, next time.

And please reopen a new PR to re-apply the commit 84d557e1a7855aa68bbaae68238b3eaf2c7487d3 you reverted by error.